python_modules/libraries/dagster-dg/dagster_dg/cli/init.py

```diff
@@ -50,7 +50,7 @@ def init_command(
     ).strip()
 
     workspace_path = scaffold_workspace(workspace_name)
-    workspace_dg_context = DgContext.from_config_file_discovery_and_cli_config(
+    workspace_dg_context = DgContext.from_config_file_discovery_and_command_line_config(
```

Comment:
This fn renamed because we now have a `DgCliConfig` type (due to rename from "global" -> "cli") that can actually be sourced from either the command line _or_ a config file. "command line config" communicates a bit more that the function is referring to parameters that originate from the command line.

Reply:
this is quite the mouthful. Do we need the first "config"?